Gemfile

```diff
@@ -42,7 +42,7 @@ group :test do
 end
 
 group :acceptance do
-  gem 'beaker', '~> 2.2'
+  gem "beaker", :git => 'git://github.com/puppetlabs/beaker.git', :ref => '7d0cccf22cc7289113de862fe4a4494872f574bb'
```

Comment:
@ajroetker #1317 makes this change unneeded, since the beaker SHA can be set in the BEAKR_VERSION env var. I will delete this commit.